codegen/Cargo.toml

```diff
@@ -9,11 +9,11 @@ version = "0.2.8"
 
 [build-dependencies.serde_codegen]
 optional = true
-version = "0.8.0"
+version = "0.8.9"
```

Comment:
`0.8.9` was the latest version I could get to compile.  Later ones gave errors like this, even after deleting cargo.lock files and cleaning the cargo cache:

```bash
~/Documents/rusoto/codegen$ cargo build
    Updating registry `https://github.com/rust-lang/crates.io-index`
error: failed to select a version for `serde_codegen` (required by `rusoto_codegen`):
all possible versions conflict with previously selected versions of `serde_codegen`
  version 0.8.9 in use by serde_codegen v0.8.9
  possible versions to select: 0.8.15, 0.8.14
```